lib/ClangImporter/ClangImporter.cpp

```diff
@@ -513,19 +513,6 @@ getNormalInvocationArguments(std::vector<std::string> &invocationArgStrs,
     if (!triple.isOSDarwin())
       invocationArgStrs.insert(invocationArgStrs.end(),
                                {"-fobjc-runtime=ios-7.0"});
-
-    // Define macros that Swift bridging headers use.
-    invocationArgStrs.insert(invocationArgStrs.end(), {
-          "-DSWIFT_CLASS_EXTRA=__attribute__((__annotate__("
-          "\"" SWIFT_NATIVE_ANNOTATION_STRING "\")))",
-          "-DSWIFT_PROTOCOL_EXTRA=__attribute__((__annotate__("
-          "\"" SWIFT_NATIVE_ANNOTATION_STRING "\")))",
-          "-DSWIFT_EXTENSION_EXTRA=__attribute__((__annotate__("
-          "\"" SWIFT_NATIVE_ANNOTATION_STRING "\")))",
-          "-DSWIFT_ENUM_EXTRA=__attribute__((__annotate__("
-          "\"" SWIFT_NATIVE_ANNOTATION_STRING "\")))",
-      });
-
```

Comment:
When did we start emitting `external_source_symbol`? Are there scenarios where a newer Swift will have to process an older Swift's PrintAsObjC header?

Reply:
It's possible, but in any case before module stability it would actually *not* work to map those Objective-C declarations back to Swift ones, so I think we're okay.